src/app/send/add-edit.component.html

```diff
@@ -1,106 +1,159 @@
-<ng-container *ngIf="send">
-    <form (ngSubmit)="submit()" [appApiAction]="formPromise">
-        <div class="content" *ngIf="send">
-            <div class="inner-content">
-                <div class="box">
-                    <div class="box-header">
-                        {{'editSend' | i18n}}
+<form (ngSubmit)="submit()" [appApiAction]="formPromise">
+    <div class="content">
+        <div class="inner-content" *ngIf="send">
+            <div class="box">
+                <div class="box-header">
+                    {{title}}
+                </div>
+                <div class="box-content">
+                    <div class="box-content-row" appBoxRow>
+                        <label for="name">{{'name' | i18n}}</label>
+                        <input id="name" type="text" name="Name" [(ngModel)]="send.name" appAutofocus>
                     </div>
-                    <div class="box-content">
-                        <div class="box-content-row" appBoxRow>
-                            <label for="name">{{'name' | i18n}}</label>
-                            <input id="name" type="text" name="Name" [(ngModel)]="send.name" appAutofocus>
+                </div>
+            </div>
+            <div class="box">
+                <div class="box-content">
+                    <ng-container *ngIf="!editMode">
+                        <div class="box-content-row box-content-row-radio">
+                            <label class="radio-header">{{'whatTypeOfSend' | i18n}}</label>
+                            <div class="item" *ngFor="let o of typeOptions">
+                                <input type="radio" class="radio" [(ngModel)]="send.type" name="Type_{{o.value}}"
+                                    id="type_{{o.value}}" [value]="o.value" (change)="typeChanged(o)"
+                                    [checked]="send.type === o.value">
+                                <label class="label-hack" for="type_{{o.value}}">
+                                    {{o.name}}
+                                </label>
+                            </div>
                         </div>
-                        <div class="box-content-row" appBoxRow *ngIf="send.type === sendType.File">
+                        <div *ngIf="send.type === sendType.File" class="box-content-row" appBowRow>
+                            <label for="file">{{'file' | i18n}}</label>
+                            <input type="file" id="file" class="form-control-file" name="file" required
+                                [disabled]="disableSend">
```

Comment:
It looks like this is the only place the disable send policy is being honored (and working). It's OK to ignore, I'll get to it once the feature is merged in, but it might make sense to remove these references until they're all implemented

Reply:
Disable Send hasn't been considered much yet so far (it's in the "not yet ready" list). Some inputs make use of `disableSend` just from how they were added to the form (whether I wrote them out manually or copy/pasted & adjusted), but that is something I'll implement fully soon another PR.

Reply:
sick thanks!